mesh/exampleapi/views.py

```diff
@@ -12,4 +10,36 @@ def index(request):
 def hello_world(request):
     if request.method == "GET":
         print(request)
-        return HttpResponse("Hello World!")
\ No newline at end of file
+        return HttpResponse("Hello World!")
+    
+# This if for testing purposes and cleanup saved images from the testing requests
+# warning: only use this if you know what you're doing with backblaze buckets
+def check_buckets(request):
```

Comment:
Why is this in example?

Reply:
Oh, I was testing the buckets to check if it works and I didn't know where else to put it

Reply:
Is this used anywhere? And if it isn't why does it need to exist if there needs to be a warning about using it?